src/plugins/intel_npu/src/plugin/npuw/llm_infer_request.hpp

```diff
@@ -42,8 +42,11 @@ class LLMInferRequest final : public ov::ISyncInferRequest {
                         ov::SoPtr<ov::ITensor> attention_mask,
                         ov::SoPtr<ov::ITensor> position_ids);
 
+    void infer_tail_mm(ov::SoPtr<ov::ITensor> output_embed);
+
     std::shared_ptr<ov::IAsyncInferRequest> m_kvcache_request;
     std::shared_ptr<ov::IAsyncInferRequest> m_prefill_request;
+    std::optional<std::shared_ptr<ov::IAsyncInferRequest>> m_tail_mm_request_opt;
```

Comment:
`optional<shared_ptr>` looks excess, your shared_ptr can be just `nullptr` if you don't have one

Reply:
I agree, but it will be counter-intuitive as we also use `shared_ptr<ov::Model>` for two another models, that are not optional. To highlight that exactly tail is optional I added `std::optional` just for transparency, that it can be `nullptr` somewhere and it is okay!

Reply:
Just a comment solves this issue.

Reply:
just use a plain shared pointer.

Reply:
Fixed, thanks!